package.json

```diff
@@ -56,7 +56,8 @@
       "node_modules",
       "coverage",
       "migrations",
-      "test"
+      "test",
+      "bin"
```

Comment:
it is location of CLI tool, it should be checked

Reply:
Ok, but Codeconv will not pass, as the formatting process has touched parts of the code that aren't tested (such as `bin`)